dashboard/app/views/levels/_apps_dependencies.html.haml

```diff
@@ -27,9 +27,9 @@
 
 -# Droplet script dependencies (only when editor present)
 - if use_droplet && !hide_source
-  %script{src: "#{static_asset_base_path}js/ace/ace.js"}
-  %script{src: "#{static_asset_base_path}js/ace/mode-javascript.js"}
-  %script{src: "#{static_asset_base_path}js/ace/ext-language_tools.js"}
+  %script{src: asset_path('js/ace/ace.js')}
+  %script{src: asset_path('js/ace/mode-javascript.js')}
+  %script{src: asset_path('js/ace/ext-language_tools.js')}
```

Comment:
I vaguely remember something about Ace depending on loading dependencies from relative paths. If this still works under the new asset path, that seems fine!

Why not `minifiable_asset_path`?

Reply:
I thought we couldn't _digest_ ace assets (not minify) precisely because ace does dynamic loading of its own assets via relative paths without knowledge of our rails asset pipeline digest system. It sounds like you've done some testing around exactly this, so maybe I'm misremembering. Still, I'd love to understand exactly how this works after this change.  Maybe some Network tab before/after comparisons would clear things up?